### ISSUE TITLE
Pypi Project links are broken, Read the Docs not updated

### ISSUE THREAD

**Issue by dkirkham** [jazzband/wagtailmenus#488]
The Project Links on PyPi - https://pypi.org/project/wagtailmenus/4.0/ - point to the wrong URLs:

Homepage: https://github.com/jazzband/wagtailmenus/tree/stable/4.0.x
Download: https://github.com/jazzband/wagtailmenus/tarball/v4.0

It appears this has been a problem in earlier releases.

I'm not sure if this should be addressed via legacy `setup.py`  or another packaging mechanism.

Also it appears the [Read The Docs site](https://wagtailmenus.readthedocs.io/en/stable/) is not showing the 4.0 docs.

**Comment by Ash-Crow** [comment]
Is there a way to access the 4.0 docs until the problem is fixed?

**Comment by MrCordeiro** [comment]
I think this has to do with the Jazzband's bot. Perhaps @jezdez can help as he is the sole roadie in the group. Another option is as @ababic for support. As the original author, he still has admin access to both readthedocs and PyPI pages.

**Comment by CARocha** [comment]
Hi all, actually used wagtail 6.x and wagtailmenus 4 but i have a error?
first warning:
wagtailmenus.FlatMenu: (wagtailadmin.W002) FlatMenu.content_panels will have no effect on snippets editing
	HINT: Ensure that FlatMenu uses `panels` instead of `content_panels`or set up an `edit_handler` if you want a tabbed editing interface.

and then when I try the admin receive this error:
File "/Users/user/projects/django/mysite/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
TypeError: 'dict' object is not callable

I followed the docs https://wagtailmenus.readthedocs.io/en/stable/installation.html but in my setting i don't put this
'wagtail_modeladmin',          # if Wagtail >=5.1; Don't repeat if it's there already
'wagtail.contrib.modeladmin',  # if Wagtail <5.1;  Don't repeat if it's there already

because i'm in version 6.1 of wagtail?
any suggestion for solved this problem? anybody?

**Comment by MrCordeiro** [comment]
> Hi all, actually used wagtail 6.x and wagtailmenus 4 but i have a error? first warning: wagtailmenus.FlatMenu: (wagtailadmin.W002) FlatMenu.content_panels will have no effect on snippets editing HINT: Ensure that FlatMenu uses `panels` instead of `content_panels`or set up an `edit_handler` if you want a tabbed editing interface.

We are treating that warning as a false positive. We are using issue #464 to keep track of it.
 Please ignore the warning and use `FlatMenu` in the way the docs tell you to.

If the problem still persists, please create a new issue for it, as we are using this issue to keep track of the project links problem.

**Comment by CARocha** [comment]
@MrCordeiro hello ok, but this error https://pastebin.com/9ZWdqBnt
how to solve I can't create any menu? is incompatible wagtailmenus with wigtail 6.1?

**Comment by Ash-Crow** [comment]
Hi, it has been six months and the documentation for the current version is still not accessible... Is there any way I can help?

**Comment by MrCordeiro** [comment]
> Hi, it has been six months and the documentation for the current version is still not accessible... Is there any way I can help?

Yes. As I said, only Jazzband roadies have permissions to fix that. If you could get @jezdez 's attention, we might be able to solve this.

**Comment by sheepman4267** [comment]
Perhaps someone affiliated with the project could [open an issue on the roadies repo](https://github.com/jazzband/help/issues/new/choose). I'd do it, but I don't think anyone over there will know who I am.

**Comment by MrCordeiro** [comment]
I tried emailing roadies@jazzband.co. Let's see what comes out of it.

**Comment by Ash-Crow** [comment]
It's been nearly a year now and the issue is still not resolved. If the roadie here can't relaunch a failed command within such a delay, would it be possible to move this package to another organization, eg. [Wagtail Nest](https://github.com/wagtail-nest)?

**Comment by MrCordeiro** [comment]
I'm game, @Ash-Crow . 

There are some steps we must do first:
- Get the ok from the other project lead, @schlich 
- It would be nice to get @ababic 's blessing too as he's the original author and still have some of the accesses to PyPI
- Add tests against the Wagtail main branch, [nightly](https://github.com/wagtail/wagtail/wiki/Nightly-testing-of-official-plugins)
- Update the release process - currently done with the jazzband bot.
- Keep the project in sync with the [Cookiecutter project template](https://github.com/wagtail/cookiecutter-wagtail-package)
- Finally, we need to ask @jezdez to transfer out... which might land us back into the problem of his lack of responses. :-/

**Comment by ababic** [comment]
For the record, I'm happy with the project living anywhere if there are people have the will/time to maintain it 😁 It would be a slight shame, but I'd also be fine with just marking the project as 'no longer maintained' and leave it be. Nothing lasts forever, and it had a good old run!

**Comment by MrCordeiro** [comment]
@ababic , we are still maintaining it. The problem is that we can't fix the links to the docs at PyPI because we lack the access...
